packages/manager/apps/public-cloud/src/index.html

```diff
@@ -8,7 +8,11 @@
     <body class="ovhstack h-100" style="overflow: hidden;">
         <div data-ng-app="ovhStack" data-ng-strict-di class="h-100 d-flex flex-column overflow-hidden" data-ng-controller="PublicCloudController as $ctrl">
             <div>
-                <ovh-manager-navbar data-navbar-options="$ctrl.navbarOptions"></ovh-manager-navbar>
+                <ovh-manager-navbar data-navbar-options="$ctrl.navbarOptions" data-sidebar-links="$ctrl.sidebarLinks">
```

Comment:
Can you remove the `div` above ? If the `ovh-manager-navbar` isn't a `block`, you could `class="d-block"`